docs/cff_to_rst.py

```diff
@@ -21,41 +29,124 @@ def parse_cff(filename: str) -> dict[str, Union[str, list[dict[str, str]]]]:
     -------
     `dict`
         A dictionary containing the parsed data from :file:`CITATION.cff`.
-
-    Raises
-    ------
-    `yaml.YAMLError`
-        If :file:`CITATION.cff` cannot be parsed.
     """
     with pathlib.Path(filename).open() as stream:
-        try:
-            return yaml.safe_load(stream)
-        except yaml.YAMLError as e:
-            print(e)  # noqa: T201
+        return yaml.safe_load(stream)
```

Comment:
By raising the exception directly, we won't risk hiding any errors resulting from this file.